src/App.jsx

```diff
@@ -1,104 +1,27 @@
 import './App.scss';
```

Comment:
Consider moving the styles from `App.scss` to a component-specific file, such as `PostList.scss`, as suggested in the previous review.